internal/pkg/ecs/ecs.go

```diff
@@ -267,48 +238,6 @@ func (c Client) StopDefaultClusterTasks(familyName string) error {
 	return c.ecsClient.StopTasks(taskIDs, ecs.WithStopTaskReason(taskStopReason))
 }
 
-func secretNameSpace(secret string) (string, error) {
-	// A secret value can be a SSM Parameter Name/ SSM Parameter ARN/ Secrets Manager ARN
-	// Note: If there is an error while parsing the secret value, this functions assumes it to be SSM Parameter.
-	// Refer to https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Secret.html
-	parsed, err := arn.Parse(secret)
-	if err != nil {
-		return ssm.Namespace, nil
-	}
-	switch parsed.Service {
-	case ssm.Namespace, secretsmanager.Namespace:
-		return parsed.Service, nil
-	default:
-		return "", fmt.Errorf("invalid ARN: not an SSM or Secrets Manager ARN")
-	}
-}
-
-// DecryptedSecrets returns the decrypted parameters from either SSM parameter store or Secrets Manager.
-func (c Client) DecryptedSecrets(secrets []*ecs.ContainerSecret) ([]EnvVar, error) {
```

Comment:
Why do we move this function to the CLI pkg 🤔

Reply:
it felt weird that the ecs package was taking a dependency on ssm and secrets manager 🤔 especially with us wanting to have split control over which session to use for each of them